etna/metrics/utils.py

```diff
@@ -74,7 +140,7 @@ def aggregate_metrics_df(metrics_df: pd.DataFrame) -> Dict[str, float]:
     if "fold_number" in metrics_df.columns:
         metrics_dict = (
             metrics_df.groupby("segment")
-            .mean()
+            .apply(lambda x: x.mean(skipna=False, numeric_only=True))
```

Comment:
If we use `.mean()` it works like `nanmean` and ignores folds with missing values. We probably don't want that to retain comparability between segments. 
However, if there is only one value in some fold, the metric can be computed. This situation is very close to not having any value in segment but the result is very different. May be it isn't good.
So, may be we could provide some other methods for aggregation by folds (like nanmean, etc).

Reply:
It seems ok to ignore missing values by default when aggregating folds. User agreed to do so when allowed this behavior on the metric level. Sure, it comes at a cost of lacking comparability, but our only alternative to retain full comparability, as I can see right now, is to raise an error on missing values. Which user also can do by not allowing missing values.